src/ws_distance.F90

```diff
@@ -0,0 +1,229 @@
+!-*- mode: F90 -*-!
+!                                                            !
+! Copyright (C) 2016 Lorenzo Paulatto                        !
+!                                                            !
+! This file is distributed under the terms of the GNU        !
+! General Public License. See the file `LICENSE' in          !
+! the root directory of the present distribution, or         !
+! http://www.gnu.org/copyleft/gpl.txt .                      !
+!                                                            !
+!------------------------------------------------------------!
+
+module w90_ws_distance
+!     if( allocated( wdist_ndeg ) ) then
+!        deallocate( wdist_ndeg, stat=ierr  )
+!        if (ierr/=0) call io_error('Error in deallocating wdist_ndeg in param_dealloc')
+!     end if
+!     if( allocated( wdist_shiftj_wsi ) ) then
+!        deallocate( wdist_shiftj_wsi, stat=ierr  )
+!        if (ierr/=0) call io_error('Error in deallocating wdist_shiftj_wsi in param_dealloc')
+!     end if
+
+  use w90_constants, only : dp
+
+  implicit none
+
+  private
+  !
+  public :: ws_translate_dist, clean_ws_translate
+  !
+  ! The number of unit cells to shift WF j to put its centre inside the Wigner-Seitz
+  ! of wannier function i. If several shifts are equivalent (i.e. they take the function
+  ! on the edge of the WS) they are all listed
+  integer, public, save, allocatable :: wdist_shiftj_wsi(:,:,:,:,:)!(3,ndegenx,num_wann,num_wann,nrpts)
+  ! The number of equivalent shifts (see above)
+  integer, public, save, allocatable :: wdist_ndeg(:,:,:)!(num_wann,num_wann,nrpts)
+  !
+  logical, save, public :: use_ws_distance = .true.
```

Comment:
Can you add this as an input parameter, or at least set it by default to `.false.` so we can merge and do a few more tests locally before making this the default behaviour? Thanks!